ctpm/init.go

```diff
@@ -65,6 +70,42 @@ const libTemplate = `#include <iostream>
 void hello() {
 	std::cout << "Hello c3pm!" << std::endl;
 }`
+const readMeTemplate = `
+
+A new C++ project.
+
+## Getting Started
+
+This project is a starting point for a C++ project.
+
+A few helpful commands to get you started if this is your first time using c3pm:
+
+### Building your project
+` + "```" + `shell
+$ ctpm build
+` + "```" + `
+
+### Add a package
+` + "```" + `shell
+$ ctpm add <package>
+` + "```" + `
+
+### Publishing your project
+` + "```" + `shell
+$ ctpm publish
+` + "```" + `
+
+<br />
+
+For help getting started with c3pm, view our
+[online documentation](https://docs.c3pm.io/), which offers tutorials, samples and
+a list of all available commands.
+`
+
+func generateReadMe(pc *config.ProjectConfig) error {
+	projectName := `# ` + pc.Manifest.Name
+	return ioutil.WriteFile(filepath.Join(pc.ProjectRoot, "README.md"), []byte(projectName+readMeTemplate), 0644)
```

Comment:
We should use the go template system instead of adding strings. Just like other templates

Reply:
I don't have the project name in the template. How can I have add it to the README without adding strings in the function ?